File: Bookmark_sorter/Bookmark_sorter/CBookmarkSorter.vb
Public Class CBookmarkSorter
    Public Sub SortBookmark(ByVal vstrBookmark As String)
        Dim oFile As CFileManager
        oFile = New CFileManager
        Dim intCount As Integer
        intCount = vstrBookmark.IndexOf("fanfiction.net")
        If intCount >= 0 Then
            oFile.SaveFF_net(vstrBookmark)
        ElseIf vstrBookmark.IndexOf("addventure") >= 0 Then
            oFile.SaveAddv(vstrBookmark)
        Else
            oFile.SaveOtherBkmk(vstrBookmark)

        End If
    End Sub
End Class

File: Bookmark_sorter/Bookmark_sorter.ConvertedToC#/CBookmarkSorter.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Bookmark_sorter
{
	public class CBookmarkSorter
	{
		public void SortBookmark(string vstrBookmark)
		{
			CFileManager oFile = null;
			oFile = new CFileManager();
			int intCount = 0;
			intCount = vstrBookmark.IndexOf("fanfiction.net");
			if (intCount >= 0) {
				oFile.SaveFF_net(vstrBookmark);
			} else if (vstrBookmark.IndexOf("addventure") >= 0) {
				oFile.SaveAddv(vstrBookmark);
			} else {
				oFile.SaveOtherBkmk(vstrBookmark);

			}
		}
	}
}
